src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewControl.cs

```diff
@@ -592,7 +592,13 @@ private void CalculatePageInfo()
 
     private void DrawMessage(Graphics g, Rectangle rect, bool isExceptionPrinting)
     {
-        using var brush = SystemColors.ControlText.GetCachedSolidBrushScope();
+        Color brushColor = SystemColors.ControlText;
+        if (SystemInformation.HighContrast && Parent is not null)
```

Comment:
User pattern matching in order to access the Parent property only once - 
```suggestion
        if (SystemInformation.HighContrast && Parent is Control parent)
```